src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Metadata/Models/ContentPartFieldDefinitionExtensions.cs

```diff
@@ -0,0 +1,18 @@
+namespace OrchardCore.ContentManagement.Metadata.Models
+{
+    public static class ContentPartFieldDefinitionExtensions
+    {
+        public static bool IsNamedPart(this ContentPartFieldDefinition fieldDefinition)
+            => fieldDefinition.ContentTypePartDefinition.Name != fieldDefinition.PartDefinition.Name;
+
+        /// <summary>
+        /// Returns full field name, if field belongs to named part technical name is used.
+        /// </summary>
+        /// <param name="fieldDefinition"></param>
+        /// <returns></returns>
+        public static string GetFullName(this ContentPartFieldDefinition fieldDefinition)
```

Comment:
Seems you miss this change

Reply:
I've placed it here because there are already other extensions located here.

![image](https://github.com/OrchardCMS/OrchardCore/assets/6403130/592a0c94-217f-46c9-9d42-4f0fba4b64d0)

Reply:
I see, we might need to move them later